mcux/CMakeLists.txt

```diff
@@ -61,6 +61,12 @@ add_subdirectory_ifdef(
   middleware/wireless/framework_5.3.3
   )
 
+if (${MCUX_DEVICE} MATCHES "MIMX8MN6")
+	zephyr_library_sources_ifdef(CONFIG_IPM_IMX_MCUX	drivers/imx8/fsl_mu.c)
+	zephyr_library_sources(drivers/imx8/fsl_gpc.c)
```

Comment:
fsl_gpc.c and fsl_mu.c should be handled in `mcux/drivers/imx8/CMakeLists.txt` not here.